subsys/bluetooth/host/classic/shell/bredr.c

```diff
@@ -1730,6 +1730,38 @@ static int cmd_set_sniff_mode(const struct shell *sh, size_t argc, char *argv[])
 }
 #endif
 
+static int cmd_get_class_of_device(const struct shell *sh, size_t argc, char *argv[])
+{
+	int err;
+	uint32_t cod;
+
+	err = bt_br_get_class_of_device(&cod);
+	if (err) {
+		shell_error(sh, "fail to set cod (err %d)", err);
+	} else {
+		shell_print(sh, "get cod:0x%04x success", cod);
+	}
+
+	return 0;
+}
+
+static int cmd_set_class_of_device(const struct shell *sh, size_t argc, char *argv[])
+{
+	int err;
+	uint32_t cod;
+
+	cod = strtoul(argv[1], NULL, 16);
+
+	err = bt_br_set_class_of_device(cod);
+	if (err) {
+		shell_error(sh, "fail to set cod (err %d)", err);
+	} else {
+		shell_print(sh, "set cod:0x%04x success", cod);
```

Comment:
The size of cod should be 3 octets.
```suggestion
		shell_print(sh, "set cod:0x%06x success", cod);
```